release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1239,6 +1239,9 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, the Kubernetes scheduler could skip scheduling certain pods for a node that received multiple restart operations. {product-title} {product-version} counteracts this issue by including the `KubePodNotScheduled` alert for pods that cannot be scheduled within 30 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-2260[*OCPBUGS-2260*])
 
+* Previously, if more than one extra label was defined for Thanos Ruler, then the statefulset could enter a recreation loop because prometheus-operator would not add the labels in a specified order each time it reconciled the custom resource. After this fix, prometheus-operator now sorts extra labels before adding them to the statefulset. (link:https://issues.redhat.com/browse/OCPBUGS-6055[*OCPBUGS-6055*])
```

Comment:
```suggestion
* Previously, if more than one label was defined for Thanos Ruler, then the statefulset could enter a recreation loop because the `prometheus-operator` did not add the labels in a specified order each time it reconciled the custom resource. After this fix, the `prometheus-operator` now sorts extra labels before adding them to the statefulset. (link:https://issues.redhat.com/browse/OCPBUGS-6055[*OCPBUGS-6055*])
```